v2/tools/generator/internal/codegen/pipeline/report_resource_versions.go

```diff
@@ -29,8 +29,12 @@ import (
 	"github.com/Azure/azure-service-operator/v2/tools/generator/internal/reporting"
 )
 
-// ReportResourceVersionsStageID is the unique identifier of this stage
-const ReportResourceVersionsStageID = "reportResourceVersions"
+const (
+	// ReportResourceVersionsStageID is the unique identifier of this stage
+	ReportResourceVersionsStageID = "reportResourceVersions"
+	v1betaVersionPrefix           = "v1beta"
```

Comment:
Do we still need this and its usage in `supportedFrom` method below?

Reply:
You're right I don't think we do. Removed.